css_day_3/style.css

```diff
@@ -0,0 +1,359 @@
+@font-face{
+  font-family: 'utm-avo';
+  src: url('font/utm-avo.woff') format('woff');
+  src: url('font/utm-avo.woff')  format('truetype');
+}
+@font-face{
+  font-family: 'utm-avo-bold';
+  src: url('font/utm-avobold.woff') format('woff'), 
+  url('font/utm-avobold.woff')  format('truetype');
+}
+*{
+  margin: 0px;
+  padding: 0px;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Apply font-family to the `body`.
```js
body {
  font-familly: ...
}
```
Anh be careful with `*` selector, this applies to every single element.